karapace/schema_reader.py

```diff
@@ -71,25 +87,44 @@ def parse_avro(schema_str: str):  # pylint: disable=inconsistent-return-statemen
         except (SchemaParseException, JSONDecodeError, TypeError) as e:
             raise InvalidSchema from e
 
+    @staticmethod
+    def parse_protobuf(schema_str: str):
+        try:
+            ts = TypedSchema(parse_protobuf_schema_definition(schema_str), SchemaType.PROTOBUF, schema_str)
+            return ts
+        except (
+            TypeError, SchemaError, AssertionError, ProtobufParserRuntimeException, IllegalStateException,
+            IllegalArgumentException, ProtobufError, ProtobufException, ProtobufSchemaParseException
+        ) as e:
+            log.exception("Unexpected error: %s \n schema:[%s]", e, schema_str)
```

Comment:
Question, is this the correct behavior? Shouldn't the exception be propagated? If not, this is missing an explicit `return None`

Reply:
yes we